### ISSUE TITLE
F.dropout的问题

### ISSUE THREAD

**Issue by JunnYu** [rixwew/pytorch-fm#21]
import torch.nn.functional as F
代码中如果需要使用F.dropout,需要加个training=self.training
F.dropout(attn_scores, p=0.1)改成F.dropout(attn_scores, p=0.1,training=self.training)
这样model.train()可以使用dropout，model.eval()不使用dropout。

**Comment by rixwew** [comment]
Thank you for reporting the bug.
I fixed it with 8ac848d
